solr/core/src/java/org/apache/solr/cluster/placement/plugins/AffinityPlacementFactory.java

```diff
@@ -350,15 +350,15 @@ private void verifyDeleteCollection(
           if (cluster.getCollection(primaryName) != null) {
             // still exists
             throw new PlacementModificationException(
-                "colocated collection "
+                "collocated collection "
```

Comment:
Should be colocated, one `l`

Reply:
Making these change across the code base.    We do use `collocated` in the changes.txt, so I went and updated that as well..